guides/common/modules/con_host-global-status-overview.adoc

```diff
@@ -0,0 +1,43 @@
+[id="host-global-status-overview_{context}"]
+= Host Global Status Overview
+
+The global status represents the overall status of a particular host.
+The status can have one of three possible values: *OK*, *Warning* or *Error*.
+You can find global status on the Hosts Overview page.
+The status displays as a small icon next to host name and has a color that corresponds with the status.
```

Comment:
```suggestion
The status displays a small icon next to host name and has a color that corresponds with the status.
```

Maybe `color that indicates the status`?

Reply:
Much of a muchness with corresponds with v indicates. I think corresponds with sounds better?